hack/community/generate.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Generate the new community manifests, given the current wmco manifests.
+
+# Must be ran within wmco root.
+
+# Example:
+# Update community manifests v5.0.1
+# Run: bash ./hack/community/generate.sh
+
+# Install tools
+echo "## Install yq"
+#curl -L https://github.com/mikefarah/yq/releases/download/v4.13.5/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+echo "   yq installed"
+
+OPERATOR_MANIFESTS="bundle/manifests"
+OPERATOR_METADATA="bundle/metadata"
+CSV="${OPERATOR_MANIFESTS}/windows-machine-config-operator.clusterserviceversion.yaml"
+FILE=../community-operators-prod/operators/community-windows-machine-config-operator/${OPERATOR_VERSION}
```

Comment:
`OPERATOR_VERSION` may be unset. Do you expect `OPERATOR_VERSION` as an env var?